source/features/admin-2fa.html.md.erb

```diff
@@ -49,14 +49,9 @@ With this method we can include or exclude users from other organisations, it's
 
 ## Reset 2FA
 
-Admins can reset 2FA for other admins who are part of their organisation. 
-
-If an admin user needs to reset their 2FA:
-
-1. The admin user can ask a team member to do it for them. 
-1. The team member needs to sign into their GovWifi admin account, go to the ‘Team members’ section’ and select ‘Reset 2FA’ next to the user’s name. 
-1. The admin user can then follow the link in the email they receive, and follow the instructions to set up 2FA again. 
+Admins can reset 2FA for other admins who are part of their organisation. They can do this from the 'Team members' section of their GovWifi admin account. 
 
 If this does not work, or if the admin does not have a team member who can do this for them, we have a [manual process](https://docs.google.com/document/d/1WDG-owrxYe58GDSjkxco1u7E-EG5_t7HLCaHT2udIvw/edit#heading=h.dr97twgaw0tc). This involves the GovWifi team setting up a video call with the admin and asking to see some ID. This is to be sure it's really them trying to access GovWifi admin.
 
-There's a macro in Zendesk that you can use if an admin asks how to reset their 2FA.
+Admins may get in touch with us to ask about resetting their 2FA. There's more information about how to respond to these requests in the [Zendesk section of the Team Manual](https://govwifi-dev-docs.cloudapps.digital/zendesk-support/). 
```

Comment:
'May' is the better verb but I wondered if 'can' is more in keeping with the house style (such as it is)?